kubernetes_state/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG - kubernetes_state
 
+## 4.0.0 / 2018-10-12
+
+* [Fixed] Fix job metrics tagging on KSM 1.4+. See [#2384](https://github.com/DataDog/integrations-core/pull/2384).
+* [Fixed] include node condition value in check message. See [#2362](https://github.com/DataDog/integrations-core/pull/2362). Thanks [dwradcliffe](https://github.com/dwradcliffe).
+* [Fixed] Fix hostname override and type for status_report.count metrics. See [#2372](https://github.com/DataDog/integrations-core/pull/2372).
+* [Changed] [kubernetes_state] don't send pod phase service checks by default. See [#2354](https://github.com/DataDog/integrations-core/pull/2354).
```

Comment:
Does this warrant a major bump? I'm personally not sure, so feel free to ignore if you think it does.